iphone/Classes/TiErrorController.m

```diff
@@ -30,6 +42,7 @@ - (void)dealloc
   RELEASE_TO_NIL(messageLabel);
   RELEASE_TO_NIL(disclosureLabel);
   RELEASE_TO_NIL(error);
+  RELEASE_TO_NIL(reason);
```

Comment:
error and reason objects are autoreleased objects. These are released again here. It'll cause crash.